.travis.yml

```diff
@@ -55,6 +55,7 @@ env:
   - TEST_TARGET="ci-iris-lambda-rust TIMED=1"
   - TEST_TARGET="ci-math-classes TIMED=1"
   - TEST_TARGET="ci-math-comp TIMED=1"
+  - TEST_TARGET="paramcoq TIMED=1"
```

Comment:
Should be `ci-paramcoq`.
Also please add to gitlab YML.

Reply:
Done, thanks for the reminder! Will also update #6054